framework/src/modules/network/network.js

```diff
@@ -16,23 +16,25 @@
 
 const { getRandomBytes } = require('@liskhq/lisk-cryptography');
 const {
-	P2P,
-	EVENT_NETWORK_READY,
-	EVENT_NEW_INBOUND_PEER,
-	EVENT_CLOSE_INBOUND,
-	EVENT_CLOSE_OUTBOUND,
-	EVENT_CONNECT_OUTBOUND,
-	EVENT_DISCOVERED_PEER,
-	EVENT_FAILED_TO_FETCH_PEER_INFO,
-	EVENT_FAILED_TO_PUSH_NODE_INFO,
-	EVENT_OUTBOUND_SOCKET_ERROR,
-	EVENT_INBOUND_SOCKET_ERROR,
-	EVENT_UPDATED_PEER_INFO,
-	EVENT_FAILED_PEER_INFO_UPDATE,
-	EVENT_REQUEST_RECEIVED,
-	EVENT_MESSAGE_RECEIVED,
-	EVENT_BAN_PEER,
-	EVENT_UNBAN_PEER,
+	p2p: { P2P },
```

Comment:
You are already exporting `P2P` from lib, so you can simply use `const { P2P, events: {...`